packages/cf_workers/cf_workers.0.1.5/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Melange bindings for Cloudflare Workers"
+description: "Melange bindings for Cloudflare Workers"
+maintainer: ["Adrian Parvin Ouano"]
+authors: ["Adrian Parvin Ouano"]
+license: "LGPL-3.0-only"
+homepage: "https://github.com/adrianparvino/cf_workers"
+doc: "https://github.com/adrianparvino/cf_workers"
+bug-reports: "https://github.com/adrianparvino/cf_workers/issues"
+depends: [
+  "ocaml"
+  "melange" {>= "5.0.1"}
+  "dune" {>= "3.13" & >= "3.13"}
```

Comment:
```suggestion
  "dune" {>= "3.13"}
```
This can be fixed by removing dune from the dependencies in dune-project. It does add itself automatically.